### PR TITLE
log replay events at debug level

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

File: styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ private ReplayEvents() {
       final long lastConsumedEvent = entry.getValue();
       final SettableTime time = new SettableTime();
       if (printLogs) {
-        LOG.info("Replaying {} up to #{}", workflowInstance.toKey(), lastConsumedEvent);
+        LOG.debug("Replaying {} up to #{}", workflowInstance.toKey(), lastConsumedEvent);
```
Comment (bergman) [inline styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java]: does the `if (printLogs)` still make sense if we're logging to debug anyway?

Comment (danielnorberg) [inline styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java]: Not sure. Need to look at the call sites and understand why it `printLogs` was introduced in the first place I guess.

**Comment by danielnorberg** [comment]
@bergman `printLogs` still seems useful in this PR: https://github.com/spotify/styx/pull/225

**Comment by ulzha** [comment]
Merging, codecov failure spurious.

**Pull request merged**